script.js

```diff
@@ -0,0 +1,87 @@
+const submitForm = document.querySelector('.input-box'); // 할 것 제출
+const button = document.querySelector('button'); // 입력버튼
+const textInput = document.querySelector('input'); // 추가 할 내용
+const todolist = document.querySelector('#todo-list'); // 버튼을 눌렀을 때 추가되는 곳
+const todo = document.querySelector('.todo');
+const done = document.querySelector('.done');
+const doneList = document.querySelector('#done-list');
+var count_todolist = 0;
+var count_donelist = 0;
+
+var localStorage = [];
+
+// 폼 제출 비동기 함수
+submitForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    console.log('hi');
+    if (textInput.value) {
+        // textInput.value에 내용이 담겼으면
+        const div = document.createElement('div');
+        const span = document.createElement('span');
```

Comment:
변수의 이름을 좀 더 역할이 잘 드러나게 지으면 좋을 것 같아요. 
```suggestion
        const Todo = document.createElement('div');
        const TodoContent = document.createElement('span');
```

Reply:
더 있는 줄 모르고 방금 봤습니다! 여기도 한 번 변수 명을 잘 생각해 봐야겠네요!

Reply:
혹시 createElement 할 때는 변수명을 CamelCase로 해야하나요? 
```suggestion
        const todo = document.createElement('div');
        const todoContent = document.createElement('span');
```
가 맞는건가요?

Reply:
엘레멘트 네이밍엔 정해진 규칙이 없습니다 ㅎㅎ 본인이 편하신 대로 하시면 됩니다(다만 일관적이기만 하면 돼요 ㅎㅎ)